### ISSUE TITLE
[DesktopTimePicker] Enable possibility to show OK and Cancel buttons

### ISSUE THREAD

**Issue by simonbaas** [mui/mui-x#4596]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Summary 💡

I would like to be able to show the OK and Cancel buttons when using the **DesktopTimePicker**.

### Motivation 🔦

I can think of two cases where it would be good to be able to show the action buttons:

1. If only changing a single value (eg. hours) it is not very intuitive that the user has to click outside the picker for the value to change (or change the rest of the values).
2. If you enable the toolbar (by showToolbar) it is not very obvious for the user how proceed after changing the value in the view shown below.
![image](https://user-images.githubusercontent.com/3048277/158660462-f8d5ca0d-2586-4f0f-8998-e22de4fa43e7.png)

If there somehow could be possible to show the action buttons I think it would be more clear to the user on how to proceed in these two cases.

**Comment by flaviendelangle** [comment]
Related to https://github.com/mui/mui-x/pull/4408#issuecomment-1104916680

I agree that every picker component should be able to have an action bar, enabled by default or not.

**Comment by flaviendelangle** [comment]
Closed by #4778 
See the following example: https://codesandbox.io/s/date-and-time-pickers-forked-02tuk4?file=/src/App.tsx:0-918

```tsx
import * as React from "react";
import TextField from "@mui/material/TextField";
import { AdapterDateFns } from "@mui/x-date-pickers/AdapterDateFns";
import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
import { DesktopTimePicker } from "@mui/x-date-pickers/DesktopTimePicker";

export default function ResponsiveTimePickers() {
  const [value, setValue] = React.useState<Date | null>(
    new Date("2018-01-01T00:00:00.000Z")
  );

  return (
    <LocalizationProvider dateAdapter={AdapterDateFns}>
      <DesktopTimePicker
        label="For desktop"
        value={value}
        onChange={(newValue) => {
          setValue(newValue);
        }}
        ampm={false}
        renderInput={(params) => <TextField {...params} />}
        componentsProps={{
          actionBar: {
            actions: ["accept", "cancel"]
          }
        }}
      />
    </LocalizationProvider>
  );
}
```

(I saw a small UI problem when AM / PM is enabled, I'll create an issue :+1: )

**Comment by simonbaas** [comment]
Looks great, thanks!
